### ISSUE TITLE
add gnupg support

### ISSUE THREAD

**Issue by jedahan** [doron-cohen/antidot#63]
need to test if both the env var AND alias is needed, or just one
```
$ export GNUPGHOME="$XDG_DATA_HOME"/gnupg
$ gpg2 --homedir "$XDG_DATA_HOME"/gnupg
```
also from the wiki https://wiki.archlinux.org/index.php/XDG_Base_Directory#Supported apparently this wont work with systemd.

**Comment by sunhasgothishaton** [comment]
I think its just one of them is needed.

https://dev.gnupg.org/T3790
states that gpgconf --create-socketdir isn't needed anymore, it says that directory is created on the fly. The wiki page was saying that this was one of the problems for systemd. Hopefully that has fixed it systemd.

**Comment by sunhasgothishaton** [comment]
I had been trying this one out, as .gnupg has been recreated in $HOME for me many times

So I can create a private key in ~/.gnupg then move it to ~/.local/share/gnupg and set GNUPGHOME and it works 
I encrypted a file when the key was in ~/.gnupg and it decrypte after I moved they key and set GNUPGHOME

They only problem I came across was when I didn't have a private key in ~/.gnupg (although there were some files there), moved the files to ~/.local/share/gnupg and then created a private key. It failed while created the private key directory.

To fix this, I had to run "gpgconf --kill gpg-agent" to restart the key agent and then I could generate a private key. I don;t know if antidot can run command during migrations, yet.

This person had the same trouble during migration of ~/.gnupg
https://bugzilla.redhat.com/show_bug.cgi?id=1362572

I'd be interested in your tests regarding gnupg
